robo/rosafe/rosafe.txt

```diff
@@ -198,50 +327,460 @@ Requesting processors
    +------------------------+-----------+---------------+----------------------------------------------------------------------+
    | pp_refSPLdBS           | double    | 100           | Reference dB SPL value to correspond to input signal RMS value of 1  |
    +------------------------+-----------+---------------+----------------------------------------------------------------------+
-   | pp_bMiddleEarFiltering | boolean   | 0             | Flag to activate themiddle ear filtering                             |
+   | pp_bMiddleEarFiltering | boolean   | 0             | Flag to activate middle ear filtering                                |
    +------------------------+-----------+---------------+----------------------------------------------------------------------+
-   | pp_middleEarModel      | string    | jepsen        | Middle ear filter model (jepsenor lopezpoveda)                       |
+   | pp_middleEarModel      | string    | jepsen        | Middle ear filter model (jepsen or lopezpoveda)                      |
    +------------------------+-----------+---------------+----------------------------------------------------------------------+
    | pp_bUnityComp          | boolean   | 0             | Compensation to have maximum of unity gain for middle ear filter     |
    +------------------------+-----------+---------------+----------------------------------------------------------------------+
+  
+  The ``PreProc`` processor is instanciated with the following command launched from **eltclsh**:
 
  .. code-block:: bash
 
 	 >> eltclsh  > ::rosAFE::PreProc {name preProc upperDepName input pp_bRemoveDC 0 pp_cutoffHzDC 20 pp_bPreEmphasis 0 pp_coefPreEmphasis 0.97 pp_bNormalizeRMS 0 pp_intTimeSecRMS 0.5 pp_bLevelScaling 0 pp_refSPLdB 100 pp_bMiddleEarFiltering 0 pp_middleEarModel jespen pp_bUnityComp 0} &
 
+* ``Gammatone`` service: 
+
+  This service is used to launch the ``Gammatone`` filterbank processor
+
+  .. _tab-rosafe-Gammatone-params:
+
+  .. table:: Input parameters of the ``Gammatone`` service of RosAFE
+
+    +---------------+------------------+---------------+----------------------------------------------+
+    | Name          | Data type        | Default value | Documentation                                |
+    +===============+==================+===============+==============================================+
+    | name          | string           | gammatone     | The name of this activity                    |
+    +---------------+------------------+---------------+----------------------------------------------+
+    | upperDepName  | string           | preProc       | The name of the upper dependency             |
+    +---------------+------------------+---------------+----------------------------------------------+
+    | **fb_type**   | string           | string        | Filterbank type (gammatone or drnl)          |
```

Comment:
Is this written in bold font to highlight that it is the most important factor?
Or is it a mistake and should be written in normal text?